scripts/us_eia/opendata/process/test_data/ng.mcf

```diff
@@ -0,0 +1,17 @@
+Node: dcid:eia/ng.n3035_4.m
```

Comment:
Do we want to keep this in its original capital case?

Reply:
yes, i think that would look more familiar and might be easier for exploration in the near term (e.g. if we want to look up a particular series in the browser)

Reply:
SG.  Stuck to original case